plugins/services/src/js/containers/pod-instances/PodInstancesTable.js

```diff
@@ -301,7 +299,7 @@ class PodInstancesTable extends React.Component {
         id: instance.getId(),
         name: instance.getName(),
         address: instance.getAgentAddress(),
-        agent: instance.agent || { id: instance.agentId },
+        agentId: instance.agent ? instance.agent.id : instance.agentId,
```

Comment:
Pairing with @DanielMSchmidt we've found the place where we augment instances with `agent` info thus getting this duality. I marked the line in the `PodInstancesContainer`. Let's think how we could eliminate this.

Reply:
@nLight @DanielMSchmidt I removed the agent obj since it isn't used anywhere else and only kept the agentId to keep it consistent in all cases.